src/components/progress-circle/progress-circle.scss

```diff
@@ -0,0 +1,97 @@
+@import "variables";
+
+@import "default-theme";
+
+
+$circle-ease-in-out : cubic-bezier(0.35, 0, 0.25, 1) !default;
+$circle-duration : 5.25s !default;
```

Comment:
For the variables that aren't completely obvious (like `default circle size`), can you add a brief `/** */` comment explaining what it is?